src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/components/share-content-dialog-item/share-content-dialog-item.component.ts

```diff
@@ -34,10 +38,19 @@ export class ShareContentDialogItemComponent extends BaseComponent implements On
 
   ngOnInit() {
 
+    if (this.item.contentSchemaId && NON_VIRTUAL_CONTENT_SCHEMAS_IDS.indexOf(this.item.contentSchemaId) === -1) {
```

Comment:
Unify the virtual items thumbnail logic in a component , maybe using  BaseBrowserItemComponent